test/Microsoft.AspNet.Authentication.Test/Cookies/CookieMiddlewareTests.cs

```diff
@@ -748,12 +749,14 @@ private Task SignOutAsWrong(HttpContext context)
         [Fact]
         public async Task MapWillNotAffectChallenge()
         {
-            var server = TestServer.Create(app =>
+            var builder = new WebApplicationBuilder()
```

Comment:
That was one of my proposals. @davidfowl didn't like the resulting double or triple nesting.